src/core/client/stream/tabs/Comments/Stream/CommentsLinks/CommentsLinks.tsx

```diff
@@ -57,13 +57,44 @@ const CommentsLinks: FunctionComponent<Props> = ({
   showGoToProfile,
 }) => {
   const { renderWindow, customScrollContainer } = useCoralContext();
+
+  // Find first keyboard focusable element for accessibility
+  const getFirstKeyboardFocusableElement = useCallback(() => {
+    const container = customScrollContainer ?? renderWindow.document;
+    let firstFocusableElement: Element | null = null;
+    let counter = 0;
+    const focusableElements = container.querySelectorAll(
+      'a[href], button, input, textarea, select, details, [tabindex]:not([tabindex="-1"])'
+    );
+    while (
+      firstFocusableElement === null &&
+      counter < focusableElements.length
+    ) {
+      if (
+        !focusableElements[counter].hasAttribute("disabled") &&
+        !focusableElements[counter].getAttribute("aria-hidden") &&
+        !focusableElements[counter].getAttribute("hidden")
```

Comment:
I can only imagine how much trial and error it took to catch all these and make it work!